src/main/java/core/basesyntax/ColorsEnum.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+public class ColorsEnum {
+
+    public enum Colors {
+        BLUE,
+        RED,
+        GREEN,
+        PINK,
+        YELLOW
+    }
+}
```

Comment:
```suggestion
 public enum Color {
        BLUE,
        RED,
        GREEN,
        PINK,
        YELLOW
    }
```

class names in the singular